### PR TITLE
Add a new doc with guidance for adding new features

### PR DESCRIPTION
**Pull request by tbrowder**
Adding information to help contributors add new features to a future language version

### CODE REVIEW

**Comment by lizmat** [comment]
I marked this PR as draft as it seemed to still be collecting commits.  I suggest if you are done adding commits, we can unmark as draft and probably just merge it?

**Comment by tbrowder** [comment]
Ah, got it. I will tweak it a bit more and then un-draft it. I was worried you found some weird Markdown bug! Thanks.

File: docs/language_versions.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 For review, nothing set in sand yet (and these things should never be set in
 stone; we need to keep learning and adapting them over time).
 
+(For details on adding new versions or features in new versions,
+see [New language versions](./New-language-versions).)
```
Comment (japhb) [inline docs/language_versions.md:7]: This link path doesn't match the name of the new file (`versions` versus `features`).  Also worth making sure the filename matches the surrounding naming conventions (case, kebab, etc.).

**Comment by lizmat** [comment]
@tbrowder It's still marked as draft.  Please advise on further actions :-)

**Pull request closed without merging**